src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
I would say grouping purely utility functions under a utility module serves a purpose. I'm fine with not going through a `util` module to access these, but I would prefer the broken apart `lib`/`util` functionality goes under its own folder.